server/protocols/jmap/src/test/java/org/apache/james/jmap/methods/SetMailboxesUpdateProcessorTest.java

```diff
@@ -20,8 +20,9 @@
 package org.apache.james.jmap.methods;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
@strokyl Please review your IDE settings

If intelliJ : ask me to give you my conf, I have no such pb, otherwise ask Lyon people.